scripts/common/host-packages.sh

```diff
@@ -93,6 +93,27 @@ function dpkg_install_host_packages() {
     local dir_prefix=""
     local packages=("${@:2}")
     _dpkg_install_host_packages "$dir" "$dir_prefix" "${packages[@]}"
+    _dpkg_apt_get_status_and_fix_broken_deps
+}
+
+function _dpkg_apt_get_status_and_fix_broken_deps() {
+    _out=$(sudo apt-get check status 2>/dev/null)
+    _status=$?
+    if [ "$_status" -eq "0" ]; then
+        logSuccess "Apt cache status checked successfully"
+    else
+        logWarn "Attempting to correct broken dependencies in place"
+        sudo apt-get install -f -y
+        _out=$(sudo apt-get check status 2>/dev/null)
+        _status=$?
+        if [ "$_status" -eq "0" ]; then
+            logSuccess "Broken dependencies fixed successfully"
+        else
+            logFail "Unable to fix broken dependencies"
+            logFail apt-get check status
+            return 1
```

Comment:
This scenario is hard to check/test so I need your help because we are aware more of the code details, if by return 1 here the expected behaviour would occurs. That means fail and allow the user open a ticket within: https://github.com/replicatedhq/kURL/blob/main/scripts/install.sh#L532